src/plots/cartesian/axis_defaults.js

```diff
@@ -103,8 +116,8 @@ module.exports = function handleAxisDefaults(containerIn, containerOut, coerce,
 
     if(showLine || containerOut.ticks) coerce('mirror');
 
-    var gridColor = Lib.coerce2(containerIn, containerOut, layoutAttributes, 'gridcolor'),
-        gridWidth = Lib.coerce2(containerIn, containerOut, layoutAttributes, 'gridwidth'),
+    var gridColor = coerce2('gridcolor', lightColor(dfltColor, options.bgColor)),
```

Comment:
@etpinard (and anyone else who cares about style details - @jackparmer ?) as part of `axis.color`, I try to make the gridlines, by default, adapt to BOTH `axis.color` and the background they're on, so that they look like a partly-transparent version of `axis.color` on top of the background, without *actually* being partly transparent because that would have a few adverse effects (extra-dark gridline crossings, and eps/pdf exports would all get rasterized).

Interestingly this shows up only in one test image right now, `size_margins`, must be the only one that changes the background and has gridlines but doesn't explicitly set their color. Here's the baseline:

![size_margins](https://cloud.githubusercontent.com/assets/2678795/14243000/484f0704-fa53-11e5-8636-906c4ef304c3.png)

and here's as amended:

![size_margins_new](https://cloud.githubusercontent.com/assets/2678795/14243007/512b3cd0-fa53-11e5-85b3-705a93b0d0d4.png)

I think that's an improvement as a default, do you agree? Of course, if someone sets the background and axis colors too close together (like both black or blackish), then the gridlines will disappear, but that may be something we need to leave to the user to sort out. As it is now if someone sets the background to `#eee` the gridlines will disappear as well...